ci-default.yml

```diff
@@ -89,13 +89,13 @@ deps:
 
   # QA
   - name: pmm-qa
-    branch: main
+    branch: pmm-3.0.0
```

Comment:
can we have them the same as for other repos? `v3`?

Reply:
@vyarosh Can we?

Reply:
we use this format for RCs, that's why I'm not sure that it's a good idea.

Reply:
Your request only creates additional work: create `v3` branch then later create `pmm-3.0.0` branch from `v3` and delete `v3`. Could we avoid and just use existing branches mask? 

`pmm-3.0.0` is a permanent branch per our current codeflows, which means that it will stay after release.
both `pmm-ui-tests` and `pmm-qa` repos have same collection of branches starting from `pmm-2.20.0` and now ending with `pmm-3.0.0`, see linked screenshots: https://app.screencast.com/rGTSqdbTVOyiX and https://app.screencast.com/xwDkQLiiWgvAw 

Looks like it's a `v3` branch is unexpected since `pmm` repo have the same branches: https://app.screencast.com/pMNGVoawFGUWe So form the consistency point it's `v3` branch violates existing branch naming. As we have the following convention `pmm-x.x.x`

Reply:
Again, that's release branches which was created after RC cut. at the same time we always had `main` branch for  
@vyarosh are you going to keep using the branch `pmm-3.0.0` for further changes for PMM 3.1.0 after we cut PMM 3.0.0?

Reply:
@BupycHuk yes, unless we are not going to have another convention. Created `pmm-3.0.0` branch is permanent and will persist for release testing version 3.0.0 and will stay further for the compatibility testing for future versions.
I've explained my preferences:

> Your request only creates additional work: delete `pmm-3.0.0` -> create `v3` branch -> then later create `pmm-3.0.0` branch from `v3` -> delete `v3`. Could we avoid and just use existing branches mask?

This is relevant question and my personal choice is to keep `pmm-3.0.0` Do you insist on creating `v3` branch instead?

Reply:
> delete v3

This one is not going to happen, it will be `main` for PMM 3.x. That's probably the thing which creates the biggest confusion here.

> will stay further for the compatibility testing for future versions.

Will stay unchanged or will include changes for PMM 3.1.0?

Reply:
It will stay unchanged and frozen for 3.0.0 since we have relevant tests and setup corresponding to the version changes. All the changes that will changes tests or setup will be in future versions branches eg `pmm-3.1.0` ; `pmm-3.2.0`
this is solves the requirements to run full tests cycles on any version, ex: `2.30.0` contains corresponding tests and setup in `pmm-2.30.0` branches. Same approach applied to version 3.X.X.

> This one is not going to happen, it will be main for PMM 3.x. That's probably the thing which creates the biggest confusion here.

Without any additional information I made an assumption: `v3` will be relevant until release of `3.0.0` and then will be merged to `main`. And since **2.x** development stream is not the primary it will be handled in branches other then `main`. My common logic is why would the primary development steam will handled not in the `main` branch. And have `1.x` branch already.